policy-test/tests/api.rs

```diff
@@ -331,3 +319,20 @@ fn mk_admin_server(ns: &str, name: &str) -> k8s::policy::Server {
         },
     }
 }
+
+async fn retry_watch_server(
+    client: &kube::Client,
+    ns: &str,
+    pod_name: &str,
+) -> tonic::Streaming<grpc::inbound::Server> {
+    // Port-forward to the control plane and start watching the pod's admin
+    // server's policy and ensure that the first update uses the default
+    // policy.
+    let mut policy_api = grpc::PolicyClient::port_forwarded(client).await;
+    loop {
+        if let Ok(rx) = policy_api.watch_port(ns, pod_name, 4191).await {
```

Comment:
maybe worth logging errors here? if a test gets stuck in a retry loop, i would want to get the error messages out of it...